src/main/archetype/ui.apps/pom.xml

```diff
@@ -50,7 +50,7 @@
                 <extensions>true</extensions>
                 <configuration>
                     <group>${groupId}</group>
-                    <name>${artifactId}</name>
+                    <name>${appId}</name>
```

Comment:
name should also contain the suffix `.ui.apps` otherwise the content package will have the same name as the ui.content package.

Reply:
This should actually be reverted back to `${artifactId}`